modules/social_features/social_embed/src/SocialEmbedConfigOverride.php

```diff
@@ -84,8 +84,8 @@ protected function addFilterOverride($text_format, $convert_url, array &$overrid
     $config = $this->configFactory->getEditable($config_name);
     $filters = $config->get('filters');
 
-    $overrides = [];
-    $overrides[$config_name]['dependencies']['module'] = $config->get('dependencies.module');
+    $dependencies = $config->getOriginal('dependencies.module');
```

Comment:
This is necessary otherwise it will override all the embed configs and lazy won't work.